### PR TITLE
Refactoring some unsupported code related to qsub

### PR DESCRIPTION
**Pull request by raviv1995**
#### Describe Bug or Feature

Some unsupported platform code has been removed

#### Attach Test and Valgrind Logs/Output

th

### CODE REVIEW

**Review by npadole20 — CHANGES_REQUESTED** [review]

File: src/cmds/qsub.c [hunk 1]
```diff
@@ -966,1202 +764,141 @@ interactive_port(void)
...
-	if (listen(X11_comm_sock, 1) < 0) {
-		perror("qsub: listening on X11 socket failed");
-		exit_qsub(1);
-	}
-	return (X11_port_str);
+	shutdown(sock, 2);
+#ifdef WIN32
+	closesocket(sock);
+#else
+	close(sock);
```
Comment (Shrini-h) [inline src/cmds/qsub.c:783]: You wont need this line. see https://github.com/openpbs/openpbs/pull/1917/files#diff-6248e35fa28821a275554b0df12547b1R1413-R1418

File: src/include/portability.h [hunk 2]
```diff
@@ -51,5 +51,8 @@
 
 #define dlerror_reset() dlerror()
 #define SHAREDLIB_EXT "so"
+#define fix_path(char, int) NULL
+#define get_uncpath(char)   NULL
+#define critical_section()  NULL
```
Comment (Shrini-h) [inline src/include/portability.h:56]: I think the `NULL` is not needed and may be left empty

**Review by Shrini-h — CHANGES_REQUESTED** [review]

**Review by npadole20 — CHANGES_REQUESTED** [review]

File: src/cmds/qsub.c [hunk 3]
```diff
@@ -966,1202 +764,141 @@ interactive_port(void)
...
- * @param[in] s - file descriptor
+ *	block - set up to wait for a job to end.
  *
  * @return Void
+ * Exits on failre
  *
  */
 static void
-writer(int s)
+block(void)
```
[the remark this answers is not in the document]
Comment (raviv1995) [reply src/cmds/qsub.c:886]: This is used in case the block option is provided. Check line 3322

**Review by Shrini-h — APPROVED** [review]

**Comment by bayucan** [comment]
Please rebase this to the latest OSS master and see if there's any conflict.

File: src/cmds/qsub.c [hunk 4]
```diff
@@ -807,13 +623,12 @@ x11_get_authstring(void)
...
+void
 exit_qsub(int exitstatus)
 {
-#ifdef WIN32
-	/* A thread that makes qsub exit, should try and acquire the Critical Section. */
-	EnterCriticalSection(&continuethread_cs);
-#endif
+/* A thread that makes qsub exit, should try and acquire the Critical Section. */
+	critical_section();
+
```
Comment (Shrini-h) [inline src/cmds/qsub.c:631]: May be this is not necessary. please see if this can be removed

**Review by bayucan — COMMENTED** [review]
Looks good. 
@npadole20 : need your sign off on this before merge.

**Review by npadole20 — APPROVED** [review]
Changes look good to me.

**Review by bayucan — APPROVED** [review]
Looks good.

**Comment by bayucan** [comment]
Merged #2042 into master.

**Pull request merged**